packages/core/core/src/Asset.js

```diff
@@ -25,13 +25,12 @@ type AssetOptions = {
   type: string,
   code?: string,
   ast?: ?AST,
-  dependencies?: Array<Dependency>,
+  dependencies?: Array<IDependency>,
```

Comment:
Is there ever a time where one of the dependencies would not be an instance of the Dependency class? Do we expect something other than the Dependency class to implement the interface? Just wondering if the interface is necessary. Not a blocker though.

Reply:
Well... I don't think so, but flow was complaining since the types in `@parcel/types` refer to the interface not the class. Is there a better way?

Reply:
Oh, right. Not sure of another way besides pulling those classes out of core, which we didn't want to do. So yeah, this seems like the best option.